src/YPS.WebUI/Controllers/HeadmastersController.cs

```diff
@@ -5,15 +5,15 @@
 using System.ComponentModel.DataAnnotations;
 using System.Linq;
 using System.Threading.Tasks;
-using YPS.Application.Auth.Command.CreateHeadMaster;
+using YPS.Application.Headmasters.Commands.CreateHeadmaster;
 using YPS.Application.HeadMasters.Queries.CheckMasterRegisterLink;
```

Comment:
Capitalize the word master. It was capitalized, but you changed it changing the other parts of the namespase

Reply:
Dont forget to change it everywhere